documentation/builders/webapp/playlists-livestreams-podcasts.md

```diff
@@ -0,0 +1,141 @@
+# Playlists, Livestreams and Podcasts
+
+By default, the Phoniebox represents music based on its metadata like album name, artist or song name. The hierarchy and order of songs is determined by their original definition. If you prefer a specific list of songs to be played, you can use playlists (files ending with `*.m3u`). Phoniebox also supports livestreams and podcasts (if connected to the internet) through playlists.
+
+## Playlists
+If you like the Phoniebox to play songs in the pre-defined order, you can use .m3u playlists.
+
+An .m3u playlist is a plain text file that contains a list of file paths or URLs to multimedia files. Each entry in the playlist represents a single song, and they are listed in the order in which they should be played.
+
+### Structure of an .m3u playlist
+An .m3u playlist is a simple text document with each song file listed on a separate line. Each entry is optionally preceded by a comment line that starts with a '#' symbol. The actual file paths or URLs of the media files come after the comment.
+
+### Creating an .m3u playlist
+
+1. You can create an .m3u playlist using a plain text editor.
+2. Open a new text file
+3. [Optional] Start by adding a comment line to provide a description or notes about the playlist.
+4. On the following lines, list the file paths or URLs of the media files you want to include in the playlist, one per line. They must refer to true files paths on your Phoniebox. They can be relative or absolute paths.
+5. Save the file with the .m3u extensionm, e.g. `my_playlist.m3u`.
+
+```
+#EXTM3U
```

Comment:
From the description its a simple m3u file, so better remove `#EXTM3U` to not confuse the user.
I think you would not want to create extended m3u data manually ^^

Do we read them anyway?